### PR TITLE
Support POST method for Etherscan.io

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by kumavis** [comment]
lets just always POST then, less logic

**Comment by kumavis** [comment]
changed to POST all

**Comment by axic** [comment]
I do think it was more useful with an option :)

**Comment by kumavis** [comment]
easy to add in when necesary, im resistant to feature bloat

File: subproviders/etherscan.js [hunk 1]
```diff
@@ -88,13 +88,19 @@ function toQueryString(params) {
...
 }
 
-function etherscanXHR(proto, module, action, params, end) {
+function etherscanXHR(proto, get, module, action, params, end) {
   var uri = proto + '://api.etherscan.io/api?' + toQueryString({ module: module, action: action }) + '&' + toQueryString(params)
   // console.log('[etherscan] request: ', uri)
 
+  // URLs have a length in many browsers, switch to POST in that case
+  // http://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers
+  if (uri.length > 2000) {
```
Comment (axic) [inline subproviders/etherscan.js]: @kumavis this part was not taken into `master` and `eth_sendRawTransaction` still uses `GET`.

**Pull request closed without merging**